jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -197,12 +200,11 @@ public void set(String image) {
      * @param path the value to set {@code path} to
      */
     public void set(File path) {
-      this.path = path;
+      this.paths = Collections.singletonList(path);
     }
 
-    @Nullable
-    public File getPath() {
-      return path;
+    public List<File> getPaths() {
+      return path != null ? Collections.singletonList(path) : paths;
```

Comment:
Good point. I expect this will be handled properly in a later PR to have a deprecation warning. Having both `<path>` and `<paths>` is not a correct usage.

Reply:
Also, maybe we add a deprecation log here for `<path>` (unless we cover this in the followup).

Reply:
Yeah, will cover the deprecation warning in the followup.